cpg_workflows/large_cohort/sample_qc.py

```diff
@@ -8,20 +8,22 @@
 import hail as hl
 from cpg_utils import Path
 from cpg_utils.config import get_config
-from cpg_utils.hail_batch import genome_build, reference_path
+from cpg_utils.hail_batch import genome_build, image_path, query_command, reference_path
 from gnomad.sample_qc.pipeline import annotate_sex
+from hailtop.batch import Batch
+from hailtop.batch.job import Job
 
 from cpg_workflows.inputs import get_cohort
 from cpg_workflows.utils import can_reuse
 
 
 def run(
-    vds_path: Path,
-    out_sample_qc_ht_path: Path,
-    tmp_prefix: Path,
+    vds_path,
+    out_sample_qc_ht_path,
+    tmp_prefix,
 ):
     if can_reuse(out_sample_qc_ht_path, overwrite=True):
-        return hl.read_table(str(out_sample_qc_ht_path))
```

Comment:
So... this condition was never triggered (can_reuse with overwrite=True always returns False), but can we check why this would return a Table object? That won't be possible if this is run as a job, but also wasn't possible AFAIK if this is run in Dataproc..

Reply:
It looks like all of `sample_qc` was committed as one file (see PR [here](https://github.com/populationgenomics/production-pipelines/pull/136)) so I can't see why the decision to return a Table object was made. Also possibly the last change to functionality of `can_reuse` in this script is by Kat [here](https://github.com/populationgenomics/production-pipelines/pull/342)